src/components/SelectSearch/index.tsx

```diff
@@ -0,0 +1,186 @@
+import { useState, PropsWithChildren, useRef, useEffect, useMemo } from "react";
```

Comment:
We may be able to modify SearchableDropdown and to match the design instead of introducing this component